client/context/helpers.go

```diff
@@ -345,3 +348,28 @@ func (ctx CoreContext) GetNode() (rpcclient.Client, error) {
 	}
 	return ctx.Client, nil
 }
+
+func getReadableString(a interface{}) {
+	val := reflect.ValueOf(a)
+	typ := reflect.TypeOf(a)
+	fields := val.NumField()
+	for i := 0; i < fields; i++ {
+		str := typ.Field(i).Type.String()
+		switch str {
+		case "[]uint8":
+			fmt.Println(typ.Field(i).Name, ":", string(val.Field(i).Interface().([]uint8)))
```

Comment:
I think usage of `fmt.Sprintf` will make things slightly cleaner to read & understand.